### ISSUE TITLE
[v2] on theme-dark messagebar input box text color is wrong

### ISSUE THREAD

**Issue by pcvengros** [framework7io/framework7#2347]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Framework7 Version: 2.2.1.
* Platform and Target: CORDOVA.

### What you did
Using messagebar component with theme-dark is resulting in input field text color dark and unreadable. It works with standard white theme.

### Expected Behavior
When theme-dark used, make input field text white or off-white.

Example:
```
......
<body class="color-theme-orange theme-dark">
<div id="app">
...............
        <div id="view-messages" class="view tab" data-name="messages" data-route-tab-id="view-messages">
            <div class="page" id="messages-page" data-name="messages-page">
                <div class="navbar">
                    <div class="navbar-inner">
                        <div class="left sliding">
                            <a href="#" class="link icon-only e-back">
                                <i class="icon f7-icons ios-only"></i>
                                <i class="icon material-icons md-only icon-back"></i>
                            </a>
                        </div>
                        <div class="title">Messages</div>
                    </div>
                </div>
                <div class="toolbar messagebar">
                    <div class="toolbar-inner">
                        <a class="link toggle-sheet" href="#">
                            <i class="icon f7-icons ios-only">camera_fill</i>
                            <i class="icon material-icons md-only">camera_alt</i>
                        </a>
                        <div class="messagebar-area">
                            <textarea class="resizable" placeholder="Message"></textarea>
                        </div>
                        <a class="link send-link" href="#">Send</a>
                        <div class="messagebar-sheet"></div>
                    </div>
                </div>
                <div class="page-content messages-content">
                    <div class="messages">
                    </div>
                </div>
            </div>
        </div>
...............
</div>
</body>
```

**Comment by pcvengros** [comment]
I had to add to ma css explicitly:
```
.md .theme-dark .messagebar textarea {
    color: rgba(255,255,255,.87);
}
```

then it started to work.

without that only ```.md. messagebar textarea``` got applied

**Comment by nolimits4web** [comment]
Fixed with df3054eb9c75612e12f32dcc68322f1535505149
